src/søknad/forside/Forside.tsx

```diff
@@ -118,6 +123,14 @@ const Forside: React.FC<any> = ({ intl }) => {
               </KnappBase>
             </FeltGruppe>
           ) : null}
+          {toggles[ToggleName.mellomlagre_søknad] && (
+            <Knapp
+              htmlType={'button'}
+              onClick={hentMellomlagretOvergangsstønad}
+            >
+              <LocaleTekst tekst={'knapp.hentMellomlagretOvergangsstønad'} />
+            </Knapp>
+          )}
```

Comment:
Hva er grunnen til at du bruker `<Knapp/>` her og ikke `<KnappBase/>`? :)

Reply:
Hehe, jeg og Mattis skjønte ikke helt hva forskjellen var på disse - hva er `<KnappBase>`? :)

Reply:
Nei, si det! De virker nesten helt like når jeg sjekker på designsystemet. 
Vi har vel brukt mye `<KnappBase>` rundt omkring der vi har måttet definere type `
flat | hoved | standard | fare`, men jeg ser at vi også har brukt `<HovedKnapp>` direkte noen steder 🤔  Tanken er vel å bruke komponentene våre så konsist som mulig, men så lenge det funker er det jo good.
Det er også første gangen jeg ser htmlType bli definert, men dette er kanskje noe designsystemet har lagt til i komponenten 😁 

https://design.nav.no/components/knapp/technical

Reply:
Ok, tror uansett ikke denne knappen kommer til å ligge her når vi får tatt en prat med Tuva, men hun har vært litt busy på brukertesting de siste dagene, så vi slengte opp noe for å teste teknisk.